applications/luci-app-acme/Makefile

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright (C) 2017 Dan Luedtke <mail@danrl.com>
+#
+# This is free software, licensed under the Apache License, Version 2.0 .
+#
+
+include $(TOPDIR)/rules.mk
+
+LUCI_TITLE:=ACME package - LuCI interface
+LUCI_DEPENDS:=luci-base luci-app-uhttpd +acme
```

Comment:
I agree. I copied that (more or less) from the other repository and was not sure if touching it would be a good idea.

Fixed it.